Guesser/FilterTypeGuesser.php

```diff
@@ -65,7 +66,7 @@ public function guessType($class, $property, ModelManagerInterface $modelManager
 
         switch ($metadata->getTypeOfField($propertyName)) {
             case 'boolean':
-                $options['field_type']    = 'sonata_type_boolean';
+                $options['field_type']    = BooleanType::class;
```

Comment:
@core23 Sorry I didn't know that the master branch still should preserve BC compatibility with sf 2.3. I don't think it's working on 2.3, but it works with 3.0+. Should I change the target branch for the PR? Or how do you achieve 3.0+ support?